tests/integration/angular-meteor-diff-array-spec.js

```diff
@@ -101,6 +103,25 @@ describe('diffArray module', function() {
         jasmine.any(Object));
     });
 
+    it('should not call any callback if no changes were made', function() {
+      var oldCollection;
+      var newCollection;
+      oldCollection = newCollection = [
+        {_id: "a", prop: "foo"},
+        {_id: "b", outer: {inner: "bar"}}
+      ];
+
+      diffArray(oldCollection, newCollection, {
+        addedAt: addedAtSpy,
+        removedAt: removedAtSpy,
+        changedAt: changedAtSpy
+      });
+
+      expect(addedAtSpy).not.toHaveBeenCalledWith();
```

Comment:
Wasn't it should be `toHaveBeenCalled()` otherwise the test can be pass if it the callback call with something that it is not empty arguments?